craft_parts/plugins/maven_plugin.py

```diff
@@ -16,16 +16,13 @@
 
 """The maven plugin."""
 
-import os
 import re
-import xml.etree.ElementTree as ET
-from pathlib import Path
 from typing import Literal, cast
-from urllib.parse import urlparse
 
 from overrides import override
 
 from craft_parts import errors
+from craft_parts.plugins._maven_util import create_maven_settings
```

Comment:
maybe `_maven_util` and `_maven_xml` should be in a new `maven` package inside `plugins`?